### ISSUE TITLE
Rename Cargo.toml package name to `up-client-zenoh-rust`?

### ISSUE THREAD

**Issue by PLeVasseur** [eclipse-uprotocol/up-client-zenoh-rust#7]
Should we rename the package name specified in Cargo.toml to be `up-client-zenoh-rust` for consistency?

WDYT?

**Comment by evshary** [comment]
I don't have a strong opinion on that. Both are ok for me.
The reason I use `uprotocol_zenoh` is that users can know easily the package is for uprotocol and zenoh. (`up` causes some confusion)
Since the package is only used in Rust, I don't add Rust in the name.
However, if other Rust package in uProtocol will use the full project name, I think it would be better to use `up-client-zenoh-rust` for consistency.

**Comment by PLeVasseur** [comment]
@sophokles73 -- thoughts? I saw that we called it `up-rust` in the Cargo.toml, so thought "OK, let's standardize around the project names being used in the Cargo.toml."

**Comment by sophokles73** [comment]
IMHO we should definitely use the `up-rust` prefix. Regarding omitting the `-rust` suffix, I totally see @evshary's point when only focusing on this crate.

However, at the GitHub repository level it definitely makes sense to include the suffix because we will have multiple `up-client-zenoh-*` repositories for different languages. I would be fine with both names `up-client-zenoh` as well as `up-client-zenoh-rust`, with a slight tendency towards the former :-)

**Comment by PLeVasseur** [comment]
Makes sense to me. So @evshary WDYT? Change it to `up-client-zenoh`?

**Comment by evshary** [comment]
Yes, I also think `up-client-zenoh` is slightly better than `up-client-zenoh-rust`. I'll change that.
